java/vector/src/main/codegen/templates/UnionListWriter.java

```diff
@@ -194,13 +194,13 @@ public void endList() {
   @Override
   public void startList() {
     vector.startNewValue(idx());
-    writer.setPosition(vector.getOffsetBuffer().getInt((idx() + 1) * OFFSET_WIDTH));
+    writer.setPosition(vector.getOffsetBuffer().getInt(((long) idx() + 1L) * OFFSET_WIDTH));
```

Comment:
nit: By adding 1L, the result of `idx()` is automatically cast to a long, so the explicit cast can be avoided?

Reply:
Sure, you are right. I just followed the style around here. I removed the explicit cast for this and others.